tests/integration/telemetry/tracing/otelcollector/tracing_test.go

```diff
@@ -124,6 +132,45 @@ func TestProxyTracingOpenTelemetryProvider(t *testing.T) {
 		})
 }
 
+func TestGatewayTracing(t *testing.T) {
+	framework.NewTest(t).Run(func(ctx framework.TestContext) {
+		appNsInst := tracing.GetAppNamespace()
+		istioInst := *tracing.GetIstioInstance()
+		ctx.ConfigIstio().YAML(istioInst.Settings().SystemNamespace, echoGatewayTracing).ApplyOrFail(ctx)
+		ctx.ConfigIstio().YAML(appNsInst.Name(), echoGateway).ApplyOrFail(ctx)
+
+		// TODO fix tracing tests in multi-network https://github.com/istio/istio/issues/28890
+		for _, cluster := range ctx.Clusters().ByNetwork()[ctx.Clusters().Default().NetworkName()] {
```

Comment:
nit: can we loop over everything but t.Skip if the cluster is in a different network? That will be more easily searchable afterward.